### PR TITLE
Add an example to show how to create paginated result.

### PR DESCRIPTION
**Pull request by harikt**
Fixes https://github.com/atlasphp/Atlas.Orm/issues/5 partially with documentation.

We can probably come with better ways later.

Thoughts ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -372,6 +372,26 @@ You can also call `fetchRow()` or `fetchRows()` to get Row objects directly
...
+
+```php
+$query = $atlas->select(ThreadMapper::CLASS)
+    // page number
+    ->page($_GET['page'])
+    // set the number of rows per page.
+    ->paging(20);
+$result = $query->fetchRecordSet();
+
+$totalResult = clone $query;
```
Comment (harikt) [inline README.md:388]: I am not sure what should we call this object. totalQuery ?

**Comment by pmjones** [comment]
I wonder if the thing to do here is to add a new method (e.g. `MapperSelect::count($col)`)  that does all that.

**Comment by harikt** [comment]
@pmjones I am good with your suggestion. Please also see https://github.com/atlasphp/Atlas.Orm/pull/31 regarding the cloning issue.

**Comment by pmjones** [comment]
@harikt Please see the new branch "counting" with this commit: https://github.com/atlasphp/Atlas.Orm/commit/3642fcea7ef175fcfb638c2bc3e69758c3a32b2d

The fetchCount() method gives a count of all rows. I suppose it would be good to have a fetchPageCount() as well, that divides by $select->getPaging() ?

**Comment by harikt** [comment]
Thanks Paul. Looks good to me.

> I suppose it would be good to have a fetchPageCount() as well, that divides by $select->getPaging() ?

Any reason you didn't called it `getPaging` itself ?

**Comment by harikt** [comment]
One thing probably I think we could fix is the clone though. So if someone cloned it, it should not trouble them.

**Comment by pmjones** [comment]
Fixed by https://github.com/atlasphp/Atlas.Orm/commit/3642fcea7ef175fcfb638c2bc3e69758c3a32b2d

**Pull request closed without merging**